sdk/python/feast/infra/online_stores/helpers.py

```diff
@@ -1,65 +1,41 @@
+import importlib
 import struct
-from typing import Any, Dict, Set
+from typing import Any
 
 import mmh3
 
-from feast.data_source import BigQuerySource, DataSource, FileSource
-from feast.errors import FeastOnlineStoreUnsupportedDataSource
+from feast import errors
 from feast.infra.online_stores.online_store import OnlineStore
 from feast.protos.feast.storage.Redis_pb2 import RedisKeyV2 as RedisKeyProto
 from feast.protos.feast.types.EntityKey_pb2 import EntityKey as EntityKeyProto
-from feast.repo_config import (
-    DatastoreOnlineStoreConfig,
-    OnlineStoreConfig,
-    RedisOnlineStoreConfig,
-    SqliteOnlineStoreConfig,
-)
 
 
-def get_online_store_from_config(
-    online_store_config: OnlineStoreConfig,
-) -> OnlineStore:
+def get_online_store_from_config(online_store_config: Any,) -> OnlineStore:
```

Comment:
It has to be `Any` unfortunately - can't be `FeastConfigBaseModel` since pydantic doesn't understand or handle "subclass of type" as a type annotation.